docker-compose.yml

```diff
@@ -22,12 +22,15 @@ services:
 
   web:
     build: .
+    env_file:
+      - env.env
     tty: true
```

Comment:
There's no env.env file submitted, and I noticed one in the .gitignore. Should this be here? Or should this be postgres.env.env or sqlite3.env.env?

(I'm not that familiar with Docker)

Reply:
Nah, the env would contain API key information, which is why I included 2 defaults to be renamed instead

Postgres example: https://github.com/bsquidwrd/ytsm/blob/master/postgres.env.env
SQlite3 example: https://github.com/bsquidwrd/ytsm/blob/master/sqlite3.env.env

Reply:
I was gonna add a "Docker README.md" file too, but figured I'd get feedback on the initial stuff before writing up a "how to" for Docker

Reply:
Ok, I understand. I think we should use SQLite as default, and if anyone wants to use postgres, he's free to change that.

Reply:
Will do, I'll make the changes now.

Reply:
If I think about it, maybe the best solution is to have separate docker-compose.yml for sqlite and postgres. For SQLite, it's not necessary to add the 'postgres' container at all. This way, we probably won't need to use any env.env files, all the configuration will be in the yml.

Reply:
The docker-compose file is mainly a template for people to use if wanting a more "in-depth" solution, but the Dockerfile is the most important part of this. In my opinion, the docker-compose file should be renamed to something like `docker-compose.example.yml`

Reply:
I understand that, but I also want to provide a quick way to set it up, and docker-compose is the perfect way to do that.

Reply:
Okay. I added an example for people to run a container easily here https://github.com/bsquidwrd/ytsm/blob/master/Docker%20README.md

I also edited the docker-compose file, as you requested incase people want to run that https://github.com/bsquidwrd/ytsm/commit/a07434fec72f7fa7dc4bc03d75ad0de710df684e

Note: the Docker README is under the assumption you setup dockerhub for this repo so the image is available for easy running. More information on how can be found here https://docs.docker.com/docker-hub/builds/#create-an-automated-build

Reply:
I just set up the dockerhub repo, the first build should be up and running.